src/services/user/services/index.js

```diff
@@ -5,6 +5,7 @@ const RegistrationSchoolService = require('./registrationSchool');
 const UsersModelService = require('./UsersModelService');
 const UserService = require('./userService');
 const MailRegistrationLink = require('./MailRegistrationLink');
+const {RegistrationConsentService, registrationConsentServiceHooks } = require('./registrationConsent');
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5771458)